content/working_with/console/widgets/secretProviders.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Secret Providers Management
+category: Widgets
+draft: false
+---
+
+Displays all the secret providers visible to the logged-in user in the current tenant (including global secrets). 
+
+![secret-providers]( /images/ui/widgets/secret-providers.png )
+
+The widget provides the following information:
+
+* **Provider name** - The name of the secret provider.
+* **Provider type** - The type of the secret provider.
+* **Creation time**
+* **Last update time**
+
+The right column of the table allows permitted users to edit the secret provider’s values or delete it.
+
+To better understand how secret providers work in {{< param product_name >}}, go to [Secret Store page]({{< relref "developer/blueprints/spec-secretstore.md" >}}) or [Using the Secret Store page]({{< relref "working_with/manager/using-secrets.md" >}}).
```

Comment:
At https://docs.cloudify.co/staging/dev/developer/blueprints/spec-secretstore/ I don't see any specific description about secret providers.
If needed we can refer to https://docs.cloudify.co/staging/dev/cli/orch_cli/secrets-providers/.

BTW I saw that at https://docs.cloudify.co/staging/dev/working_with/manager/using-secrets/ there are Secrets widget screenshots to update.

Reply:
do you think I should also create https://docs.cloudify.co/staging/dev/working_with/manager/using-secrets/ & https://docs.cloudify.co/staging/dev/developer/blueprints/spec-secretstore/ but for secret providers store? Not sure if the scope of this ticket is only specific for widget documentation or wider? 🤔

Reply:
No, no, I don't think so. It's not on our side to create these kind of pages.

Reply:
okay, so I removed that line - https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2498/commits/b4a7b7cc6291f071c85a58ff66fe2b1d7b6d0fe3 ✔️